seed/challenges/02-javascript-algorithms-and-data-structures/basic-javascript.json

```diff
@@ -679,19 +679,19 @@
         "Change the <code>0.0</code> so that <code>quotient</code> will equal to <code>2.2</code>."
       ],
       "challengeSeed": [
-        "var quotient = 0.0 / 2.0;",
+        "var quotient = 0.0 / 2.0; // Fix this line",
         "",
         ""
       ],
       "tail": [
         "(function(y){return 'quotient = '+y;})(quotient);"
       ],
-      "solutions": [
-        "var quotient = 4.4 / 2.0;"
-      ],
+      "solutions": [],
       "tests": [
         "assert(quotient === 2.2, 'message: The variable <code>quotient</code> should equal <code>2.2</code>.');",
-        "assert(/\\//.test(code), 'message: You should use the <code>/</code> operator');"
+        "assert(/4\\.40*\\s*\\/\\s*2\\.*0*/.test(code), 'message: You should use the <code>/</code> operator.');",
```

Comment:
This regex is really close to perfect! 
`4.4 / 20` will also pass this test. It's not a big deal, as it will break the first test, so one would have to do something crazy like 
`(4.4 / 20 * 0) + 2.2` is also a way pass, but I feel like if a camper can come up with that, they understand.
Maybe we should make the test more lenient? Something like `/\d\s*\/\s*\d/` should do the trick, while allowing weird (but correct) solutions like `8.8 / 4` (which should pass, according to the message).